### PR TITLE
Calendar with Event Creation

### PR DESCRIPTION
**Pull request by justagoodguy-omega**

### CODE REVIEW

File: app/src/test/java/com/github/onedirection/CalendarFragmentTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.github.onedirection;
+
+
+import com.github.onedirection.navigation.fragment.calendar.CalendarFragment;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertThrows;
+
+public class CalendarFragmentTest {
```
Comment (Ef55) [inline app/src/test/java/com/github/onedirection/CalendarFragmentTest.java:10]: Isn't this a bit light ? (There is only/already 69% of coverage)

File: app/src/main/java/com/github/onedirection/navigation/fragment/calendar/DaySelectPopupFragment.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            @Override
+            public void onClick(DialogInterface dialog, int which) {
+                switch (which) {
+                    //add events
+                    case 0:
+                        calendar.showEventCreationScreen(day, month, year);
+                        break;
+                    //view events
+                    case 1:
+                        break;
```
Comment (Ef55) [inline app/src/main/java/com/github/onedirection/navigation/fragment/calendar/DaySelectPopupFragment.java:40]: What is a "view event" in this context ?

**Review by Ef55 — APPROVED** [review]
Otherwise it's all good.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -40,4 +40,10 @@
     <string name="start_time_text">Start time:</string>
     <string name="end_time_text">End time:</string>
 
+    <!-- Strings related to calendar and event creation -->
+    <string name= "popup_title"> "" </string>
+    <string name= "cancel"> "cancel" </string>
+    <string name= "day_select_dialog_tag">"daySelectedDialog"</string>
```
Comment (TheImpressiveDonut) [inline app/src/main/res/values/strings.xml:46]: Just so you know, you don't need to put the quotes in the value when declaring a string in xml.

File: app/src/test/java/com/github/onedirection/CalendarFragmentTest.java (hunk 1 above)
Comment (TheImpressiveDonut) [reply app/src/test/java/com/github/onedirection/CalendarFragmentTest.java:10]: +1

File: app/src/main/java/com/github/onedirection/navigation/fragment/calendar/DaySelectPopupFragment.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import android.os.Bundle;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+
+import com.github.onedirection.R;
+
+public class DaySelectPopupFragment extends DialogFragment {
+
+    final private CharSequence[] buttonTitles = {"Add Event", "View Events"};
```
Comment (TheImpressiveDonut) [inline app/src/main/java/com/github/onedirection/navigation/fragment/calendar/DaySelectPopupFragment.java:14]: You should use the array value you have declared in strings.xml for {"Add Event", "View Events"}, if possible.

**Review by TheImpressiveDonut — APPROVED** [review]
Nice

**Pull request merged**